public/app/core/components/AppChrome/MegaMenu/navBarItem-translations.ts

```diff
@@ -106,6 +106,26 @@ export function getNavTitle(navId: string | undefined) {
       return t('nav.apps.title', 'Apps');
     case 'alerts-and-incidents':
       return t('nav.alerts-and-incidents.title', 'Alerts & IRM');
+    case 'plugin-page-grafana-incident-app':
+      return t('nav.incidents.title', 'Incidents');
+    case 'plugin-page-grafana-ml-app':
+      return t('nav.machine-learning.title', 'Machine learning');
+    case 'plugin-page-grafana-slo-app':
+      return t('nav.slo.title', 'SLO');
+    case 'plugin-page-k6-app':
+      return t('nav.performance-testing.title', 'Performance testing');
+    case 'monitoring':
+      return t('nav.observability.title', 'Observability');
+    case 'plugin-page-grafana-k8s-app':
+      return t('nav.kubernetes.title', 'Kubernetes');
+    case 'plugin-page-grafana-app-observability-app':
+      return t('nav.application.title', 'Application (preview)');
```

Comment:
I assume these titles override the ones set by the plugins, so maybe they need to be informed that they need to update here too in order to get it changed. Just leaving this comment as a reminder.